src/exits/zcl_abapgit_exit.clas.abap

```diff
@@ -341,4 +341,15 @@ CLASS zcl_abapgit_exit IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD zif_abapgit_exit~validate_before_push.
+
+    IF gi_exit IS NOT INITIAL.
+      gi_exit->validate_before_push(
+          is_comment     = is_comment
+          io_stage       = io_stage
+          io_repo        = io_repo ).
```

Comment:
catch and ignore like in the other methods

`CATCH cx_sy_ref_is_initial cx_sy_dyn_call_illegal_method ##NO_HANDLER.`